deploy/examples/basic_automated_cr.yml

```diff
@@ -0,0 +1,18 @@
+# This will deploy Quay with Clair automatically on the url quay.app.example.com while keeping the configuration to update it
+
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: QuayEcosystem
+metadata:
+  name: quayecosystem
+spec:
+  quay:
+    imagePullSecretName: redhat-pull-secret
+    deploymentStrategy: Recreate
+    skipSetup: false
+    keepConfigDeployment: true
+    routeHost: quay.app.example.com
+  redis:
+    imagePullSecretName: redhat-pull-secret
```

Comment:
Typically, Redis would originate from a different registry than Quay/Clair (Red Hat Container Catalog vs Quay). By default, the operator specified _registry.access.redhat.com_ as the registry hosting Redis and this registry does not require authentication

Reply:
@scouturier bump ^^

Reply:
I have updated both files to use the red hat repo rather than quay.